src/ynEIGEN/WithdrawalsProcessor.sol

```diff
@@ -7,7 +7,7 @@ import {AccessControlUpgradeable} from
     "lib/openzeppelin-contracts-upgradeable/contracts/access/AccessControlUpgradeable.sol";
```

Comment:
This test file is now very large at 2k lines

Is it possible to separate the tests here into logical units into multiple files to help with this?